crates/config/src/zksync.rs

```diff
@@ -111,10 +116,15 @@ impl ZkSyncConfig {
             // Set in project paths.
             remappings: Vec::new(),
             detect_missing_libraries: self.detect_missing_libraries,
-            system_mode: self.eravm_extensions,
+            enable_eravm_extensions: self.enable_eravm_extensions,
             force_evmla: self.force_evmla,
-            // TODO: See if we need to set this from here
-            output_selection: Default::default(),
+            llvm_options: self.llvm_options.clone(),
+            output_selection: OutputSelection {
+                all: Some(FileOutputSelection {
+                    per_file: None,
+                    per_contract: Some([OutputSelectionFlag::ABI].into()),
```

Comment:
Nice, are we sure we don't use some fields that used to be present like the metadata? I think it would came up just as invariant testing but just wondering.

Reply:
I don't think so, we use the bytecodes but those are always output. If something comes up we can add it then